Sources/EmbraceCoreDataInternal/BackgroundTaskWrapper.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Copyright © 2025 Embrace Mobile, Inc. All rights reserved.
+//
+
+import Foundation
+
+#if canImport(UIKit) && !os(watchOS)
+import UIKit
+import EmbraceCommonInternal
```

Comment:
Wrap this with `#if !EMBRACE_COCOAPOD_BUILDING_SDK` like this:
```suggestion
#if !EMBRACE_COCOAPOD_BUILDING_SDK
import EmbraceCommonInternal
#endif
```